client/my-sites/email/email-providers-comparison/email-provider-details/index.jsx

```diff
@@ -36,6 +39,13 @@ class EmailProviderDetails extends React.Component {
 		) );
 	}
 
+	onButtonClick = ( e ) => {
+		const { onButtonClick } = this.props;
+		if ( onButtonClick ) {
+			onButtonClick( e );
+		}
+	};
+
```

Comment:
Not mandatory at all but another way to handle missing event handlers could be to use a `noop` function:

```javascript
static defaultProps = {
  onButtonClick: noop,
};

onButtonClick = () => {
  this.props.onClick();
};
```

Reply:
Yeah, this makes the code a lot cleaner.